km_api/know_me/tests/serializers/test_km_user_detail_serializer.py

```diff
@@ -33,6 +33,7 @@ def test_serialize(
         'url': url,
         'name': km_user.name,
         'quote': km_user.quote,
+        'image': km_user.image,
```

Comment:
This is not actually testing anything since `km_user` is created without an image. To fix that, you can change the test signature to look like:
```python
def test_serialize(
    api_rf,
    image,
    km_user_factory,
    profile_factory,
    serializer_context):
```

Then you will have to pass that image to the created Know Me user:
```python
km_user = km_user_factory(image=image)
```

Once you make that change, you'll find that the test is failing since the serializer actually outputs the URL of the image. To fix this, you can create a request and then build the URI of that request:
```python
image_url = api_rf.get(km_user.image.url).build_absolute_uri()

expected = {
    # stuff
    'image': image_url,
    # other stuff
}